client/lib/analytics/ad-tracking.js

```diff
@@ -1729,6 +1772,67 @@ function recordOrderInGoogleAnalytics( cart, orderId ) {
 	window.ga( 'ecommerce:addTransaction', transaction );
 }
 
+/**
+ * Records an order/sign_up in Google Ads Gtag
+ *
+ * @param {Object} cart - cart as `CartValue` object
+ * @param {Number} orderId - the order id
+ * @param {Boolean} containsJetpackPlan - whether the cart includes a Jetpack product
+ * @param {Boolean} containsNonJetpackProduct - whether the cart includes a non-Jetpack product
+ * @param {Number} jetpackCost - cost of Jetpack plan
+ * @param {Number} nonJetpackCost - cost of non-Jetpack product
+ * @returns {void}
+ */
+function recordOrderInGoogleAds(
```

Comment:
I _think_ this is currently in conflict with:

https://github.com/Automattic/wp-calypso/blob/0e0ba0887d1f65e383a717c7cd94f6367fabcb9a/client/lib/analytics/ad-tracking.js#L897-L929